lib/alephant/broker/response/base.rb

```diff
@@ -48,10 +52,19 @@ def add_no_cache_headers
           log
         end
 
+        def add_etag_allow_header
+          headers.merge!("Access-Control-Expose-Headers" => "ETag")
+        end
+
         def component_not_modified(headers, request_env)
```

Comment:
Just realised i'm feeling a little uneasy about this method... It takes `headers` as a param, but headers is an attribute of the class - therefore we have a little bit of a name collision.  But the other thing is that it's not really an instance level method as it's not driven by the state of the object - maybe we could alter this to a class level method?  This would be a little more intention revealing to it's purpose and stateless behaviour too.